src/main/java/org/maxgamer/quickshop/util/envcheck/EnvironmentChecker.java

```diff
@@ -107,12 +112,21 @@ public ResultReport run(EnvCheckEntry.Stage stage) {
                     Util.debugLog("Skip test: " + envCheckEntry.name() + ": Except stage: " + Arrays.toString(envCheckEntry.stage()) + " Current stage: " + stage);
                     continue;
                 }
-                ResultContainer executeResult = (ResultContainer) declaredMethod.invoke(this);
-                //plugin.getLogger().info("Result: "+executeResult.getResultMessage());
-                if (executeResult.getResult().ordinal() > result.ordinal()) { //set bad result if its worse than the latest one.
-                    result = executeResult.getResult();
+
+                if (!startupArgs.contains("-Dorg.maxgamer.quickshop.util.envcheck.skip." + envCheckEntry.name())) {
```

Comment:
We should check the system properties, not the args.

Reply:
No I just want some args not all, it is useless so only check input arguments